src/Bridge/Symfony/Bundle/Resources/config/oauth2.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+
+        <!-- OAuth2 config -->
+
+        <service id="api_platform.oauth2.oauth2_config"
```

Comment:
It's not a service, it's an (immutable) value object. It should not be registered here. You could use a factory but IMO it's better to inject those values directly as parameters in the constructors.